src/masternodes/rpc_loan.cpp

```diff
@@ -29,14 +29,34 @@ UniValue setLoanTokenToJSON(CLoanSetLoanTokenImplementation const& loanToken, DC
         return (UniValue::VNULL);
 
     loanTokenObj.pushKV("token",tokenToJSON(tokenId, *static_cast<CTokenImplementation*>(token.get()), true));
-    loanTokenObj.pushKV("priceFeedId", loanToken.priceFeedTxid.GetHex());
+    loanTokenObj.pushKV("priceFeedId", loanToken.priceFeed.first + "/" + loanToken.priceFeed.second);
     loanTokenObj.pushKV("interest", ValueFromAmount(loanToken.interest));
 
     UniValue ret(UniValue::VOBJ);
     ret.pushKV(loanToken.creationTx.GetHex(), loanTokenObj);
     return (ret);
 }
 
+PriceFeedPair DecodePriceFeed(const UniValue& value)
```

Comment:
Should add some unit test for this function

Reply:
All the tests do it.